dask_cuda/local_cuda_cluster.py

```diff
@@ -156,7 +156,7 @@ def __init__(
         )
         self.device_memory_limit = device_memory_limit
 
-        self.rmm_pool_size = rmm_pool_size
+        self.rmm_pool_size = rmm_pool_size or dask.config.get("rmm.pool-size")
```

Comment:
Why is this necessary? `rmm.pool-size` was not intended to be used like this and was introduced specifically to allow adding an RMM pool to the scheduler when `protocol="ucx"`, as we weren't allowed to add it as a parameter to the scheduler's class/CLI. There's two critical side-effects in doing this, specifically when using `protocol="ucx"`:

1. `rmm.reinitialize` will be called a second time for workers in https://github.com/dask/distributed/blob/44b2358e33a0738c4c70ca96db4242636245e07d/distributed/comm/ucx.py#L103-L108 -- probably of lesser concern;
1. The scheduler (and maybe the client too, but I'm not sure) will also use an RMM pool of the same size as the workers, meaning that creating a large enough pool will cause an OOM.

Reply:
@jakirkham was there a workflow you had in mind ?